### PR TITLE
Assets 1.21

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: rotkehlchen/assets/asset.py [hunk 1]
```diff
@@ -89,6 +89,15 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0x15B543e986b8c34074DFc9901136d9355a537e7E'): 'STCCOIN',
     strethaddress_to_identifier('0x8f136Cc8bEf1fEA4A7b71aa2301ff1A52F084384'): 'STC',
     'MER': 'MER',
+    # For some reason seems that XSILV and GOLD are the same asset in bittrex
```
Comment (LefterisJP) [inline rotkehlchen/assets/asset.py:92]: XSILV and GOLD or XGOLD?

File: rotkehlchen/assets/converters.py [hunk 2]
```diff
@@ -354,13 +356,15 @@
...
     'CBC',  # neither in coingecko nor cryptocompare
+    'CIND',  # neither in coingecko nor cryptocompare
     # Tether CNH. As of 30/09/2019 no data found outside of Bittrex for this token
     # https://medium.com/bittrex/new-bittrex-international-listing-tether-cnh-cnht-c9ad966ac303
     'CNHT',
     # Credit coin. As of 29/01/2020 no data found outside of Bittrex for this token
     # https://global.bittrex.com/Market/Index?MarketName=BTC-CTC
     'CTC',
     'DAF',  # neither in coingecko nor cryptocompare
+    'DATA',  # Couldn't find what token this is
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:367]: Didn't you say it's DAT after all?

File: rotkehlchen/assets/converters.py [hunk 3]
```diff
@@ -639,6 +664,18 @@
     'TT',  # delisted
 )
 
+UNSUPPORTED_GEMENI_ASSETS = (
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:667]: It's GEMINI

```suggestion
UNSUPPORTED_GEMINI_ASSETS = (
```

File: rotkehlchen/assets/converters.py [hunk 4]
```diff
@@ -663,6 +700,8 @@
 COINBASE_PRO_TO_WORLD = {v: k for k, v in WORLD_TO_COINBASE_PRO.items()}
 COINBASE_TO_WORLD = {v: k for k, v in WORLD_TO_COINBASE.items()}
 UPHOLD_TO_WORLD = {v: k for k, v in WORLD_TO_UPHOLD.items()}
+BITSTAMP_TO_WORLD = {v: k for k, v in WORLD_TO_BITSTAMP.items()}
+GEMENI_TO_WORLD = {v: k for k, v in WORLD_TO_GEMENI.items()}
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:704]:
```suggestion
GEMINI_TO_WORLD = {v: k for k, v in WORLD_TO_GEMINI.items()}
```

File: rotkehlchen/assets/converters.py [hunk 5]
```diff
@@ -880,7 +920,13 @@ def asset_from_gemini(symbol: str) -> Asset:
     """
     if not isinstance(symbol, str):
         raise DeserializationError(f'Got non-string type {type(symbol)} for gemini asset')
-    return symbol_to_asset_or_token(symbol)
+
+    if symbol in UNSUPPORTED_GEMENI_ASSETS:
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:924]:
```suggestion
    if symbol in UNSUPPORTED_GEMINI_ASSETS:
```

File: rotkehlchen/assets/converters.py [hunk 6]
```diff
@@ -880,7 +920,13 @@ def asset_from_gemini(symbol: str) -> Asset:
...
+        raise UnsupportedAsset(symbol)
+
+    name = GEMENI_TO_WORLD.get(symbol, symbol)
+
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:928]:
```suggestion
    name = GEMINI_TO_WORLD.get(symbol, symbol)
```

**Review by LefterisJP — COMMENTED** [review]
Some small comments

File: rotkehlchen/assets/converters.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/assets/converters.py:367]: Ah I have to remove it. Yes omw

File: rotkehlchen/assets/asset.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/assets/asset.py:92]: xgold

File: rotkehlchen/assets/converters.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/assets/converters.py:367]: This is for BITTREX and by searching in their markets they don't have any DATA token or `streamr` that could be the candidate. Searched again for posts and tweets but there is nothing

File: rotkehlchen/assets/converters.py [hunk 7]
```diff
@@ -880,7 +920,13 @@ def asset_from_gemini(symbol: str) -> Asset:
...
     if not isinstance(symbol, str):
         raise DeserializationError(f'Got non-string type {type(symbol)} for gemini asset')
-    return symbol_to_asset_or_token(symbol)
+
+    if symbol in UNSUPPORTED_GEMINI_ASSETS:
+        raise UnsupportedAsset(symbol)
+
+    name = GEMINI_TO_WORLD.get(symbol, symbol)
+
+    return symbol_to_asset_or_token(name)
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:929]: Look more carefully in my comments. I had also asked for removal of whitespace.

```suggestion
    name = GEMINI_TO_WORLD.get(symbol, symbol)
    return symbol_to_asset_or_token(name)
```

Don't do it now though, no need to run all tests for this again.

**Review by LefterisJP — APPROVED** [review]
LGTM.

**Pull request merged**